parameters.cpp

```diff
@@ -210,7 +210,7 @@ bool Parameters::addParameters(){
    
 // Output variable parameters
    Readparameters::addComposing("variables.output", "List of data reduction operators (DROs) to add to the grid file output. Each variable to be added has to be on a new line output = XXX. Available are (20171123) B BackgroundB PerturbedB E Rhom Rhoq populations_Rho V populations_V populations_moments_Backstream populations_moments_NonBackstream populations_MinValue populations_RhomLossAdjust LBweight MaxVdt MaxRdt MaxFieldsdt MPIrank FsGridRank FsGridBoundaryType BoundaryType BoundaryLayer populations_Blocks fSaved populations_accSubcycles VolE HallE GradPeE BackgroundBedge VolB BackgroundVolB PerturbedVolB Pressure populations_Pressure populations_PTensor derivs BVOLderivs GridCoordinates Potential BackgroundVolE ChargeDensity PotentialError SpeciesMoments MeshData");
```

Comment:
Pressure should be removed from here, right?

Reply:
No the "Pressure" identifier in the config file causes DRO::VariablePressureSolver to be added, which does what one would expect: the overall pressure of all populations.

Reply:
Date identifier needs changing to 2018. 

populations_MinValue should be changed to populations_EffectiveSparsityThreshold. 

I'd recommend switching from populations_RhomLossAdjust to populations_RhoLossAdjust.

Remove populations_Pressure and SpeciesMoments from this list as they no longer exist.

For the diagnostic listing line, there's a conflict in populations Rho(m)LossAdjust naming. Also, populations_MinDistributionFunction should then be dropped, as per our discussions.

Reply:
Yup, done!

Reply:
Do we need a separate Pressure? It is redundant information if the populations/PTensorDiagonal are stored. But OK maybe a user only wants the total pressure and not have to store the tensor's diagonal components.

Reply:
Well, the reducer exists. No point in removing working code for a potentially useful variable. We can always decide to simply not use it in a simulation.

Reply:
ok